.github/issue_template.md

```diff
@@ -0,0 +1,23 @@
+### Enhancement Description
+
+- One-line enhancement description (can be used as a release note):
+- Zarf Enhancement Proposal: <!-- link to zarf-dev/proposals file; if none yet, link to PR -->
+- Primary contact (assignee):
+- Enhancement target (which target equals to which milestone/version):
+  - Alpha release target (x.y):
+  - Beta release target (x.y):
+  - Stable release target (x.y):
```

Comment:
@salaxander @schristoff Are we planning to put features through alpha stable and beta like Kubernetes?

Reply:
Yes